drip/drips.py

```diff
@@ -65,6 +94,10 @@ def from_email(self):
     def from_email_name(self):
         return self.drip_base.from_email_name
 
+    @property
+    def from_phone_number(self):
+        return self.drip_base.from_phone_number
+
```

Comment:
SMS has a bit more limitations than Emails and the cases we faced before required rotating "from numbers" for the same text depending on the number of outbound messages

That's said, I don't think we should define it and it should be handled in the signal receiver